yarn-project/slasher/src/slasher_client.test.ts

```diff
@@ -272,6 +277,18 @@ describe('SlasherClient', () => {
     const slot2 = BigInt(Math.floor(Math.random() * 1000000));
     const payload2 = await slasherClient.getSlashPayload(slot2);
     expect(payload2).toBe(config.slashOverridePayload);
+
+    slasherClient.proposalExecuted({ round: 0n, proposal: config.slashOverridePayload.toString() });
```

Comment:
Maybe would be good for us to have the test with a duplicate, just for good measure to show that all of them are not removed etc. e.., same payload but multiple times, e.g., repeat offenders.